packages/salesforcedx-vscode-core/src/commands/forceApexClassCreate.ts

```diff
@@ -80,16 +80,24 @@ class SelectDirPath implements ParametersGatherer<{ outputdir: string }> {
     const rootPath = vscode.workspace.rootPath;
     let outputdir;
     if (rootPath) {
-      outputdir = this.explorerDir
-        ? path.relative(rootPath, this.explorerDir)
-        : await vscode.window.showQuickPick(
-            this.globDirs(rootPath, 'classes'),
-            <vscode.QuickPickOptions>{
-              placeHolder: nls.localize(
-                'force_apex_class_create_enter_dir_name'
-              )
-            }
-          );
+      const normalizedRoot = path.normalize(rootPath);
+      if (this.explorerDir) {
+        if (process.platform === 'win32') {
+          outputdir = path.normalize(this.explorerDir.substr(1));
```

Comment:
Do you know why this happens? I want to make sure that this is always the case and not just an issue with the latest version of VS Code (could it be a regression).

My fear is that we _fix_ it and it turns out to be a regression on their end that they will fix by the time the next version of VS Code goes out.